sorc/build_gefs_ensstat.sh

```diff
@@ -1,22 +1,21 @@
 #! /usr/bin/env bash
 set -eux
 
-source ./machine-setup.sh > /dev/null 2>&1
+source ./machine-setup.sh #> /dev/null 2>&1
```

Comment:
Remove these tail comments if this is the way it is going to stay

Reply:
OK. I will remove them today after the ecflow meeting. Also I want to modify the versions today.

Reply:
Just removed all tail comments and update the version/gefs.ver.